_data/other.yml

```diff
@@ -103,6 +103,13 @@ websites:
       img: commuterbenefits.png
       tfa: No
       twitter: CommuterBenefit
+      
```

Comment:
@thegeekkid if you use notepad++ you can install the editorconfig plugin. That way your edits will automatically comply with our mark-up guidelines. :wink: 
https://github.com/editorconfig/editorconfig-notepad-plus-plus#install-from-the-plugin-manager